browser/mocha/stub.js

```diff
@@ -25,20 +33,17 @@ extendInterfaces('stub', function(context, teardown) {
     var proto = document.createElement(tagName).constructor.prototype;
 
     // For all keys in the implementation to stub with..
-    var keys = Object.keys(implementation);
-    keys.forEach(function(key) {
+    var stubs = Object.keys(implementation).map(function(key) {
```

Comment:
I didn't actually change any behavior sinon always monkey patched the object in place. Again the change to map to necessary in order to keep the list of stubs rather than trying to find stubs again using getPropertyDescriptor.